src/styles.css

```diff
@@ -1,3 +1,21 @@
+.react-js-cron-select .ant-select-selection-wrap {
```

Comment:
Could you move it down after the block where we define `.react-js-cron-select` style?
I would say after line 22 of the original file (after line 40 from this PR file).